lib/gollum/assets.rb

```diff
@@ -16,7 +17,7 @@ def self.sprockets(dir = File.dirname(File.expand_path(__FILE__)))
       env.append_path ::File.join(dir, 'public/gollum/images')
       env.append_path ::File.join(dir, 'public/gollum/fonts')
 
-      env.js_compressor  = :uglify unless Precious::App.development?
+      env.js_compressor  = ::Uglifier.new(harmony: true) unless Precious::App.development?
```

Comment:
Ran into https://github.com/lautis/uglifier/issues/127 when trying to add mermaid as an asset